Makefile

```
@@ -203,8 +211,14 @@ install-deps:
 	case $(_PYTHON_VENV) in python3*) yum install gcc -y; esac
 	virtualenv --system-site-packages -p /usr/bin/$(_PYTHON_VENV) $(VENVNAME); \
 	. $(VENVNAME)/bin/activate; \
+	pip install -U pip; \
 	pip install --upgrade setuptools; \
-	pip install --upgrade -r requirements.txt
+	pip install --upgrade -r requirements.txt \
+	# In case the top commit Depends-On some yet unmerged framework patch - override master leapp with the proper version
+	if [[ ! -z "$(REQ_LEAPP_PR)" ]] && [ $(REQ_LEAPP_PR_IS_OPEN) = 'True' ]; then \
```

I am thinking now whether the second part of the condition is correct. I haven't tested it but thinking about:

```suggestion
	if [[ ! -z "$(REQ_LEAPP_PR)" ]] && [[ "$(REQ_LEAPP_PR_IS_OPEN)" == "True" ]]; then \
```

---

Yeah, you are right, will fix in next commit. Thanks a lot!

---

Should be fixed now.